inputs/wyrmroost.txt

```diff
@@ -0,0 +1,142 @@
+{
+	"prefix":"wyrmroost",
+	"prototypes":{
+	
+	},
+	"implementations":{
+        // Explicitly defined
+        // H1 one full unit of hunger S1 one full unit of saturation
+		"desert_wyrm":{
+            "vital":2,
+            "faunal":20,
+            // it's a living being in an inventory
+        },
+        "cooked_desertwyrm":{
+            "vital":1,
+            "faunal":20,
+            // it's a cooked being in an inventory H5 S10
+        },
+        "coin_dragon":{
+            "vital":5,
+            "faunal":15,
+            "regal":8,
+            "metal":4,
+                        // it's a living being in an inventory (dragon with a golden nugget)
+        },
+        "blue_geode":{
+            "crystal":200,
+            "weird":15,
+            //cheaper diamond like item with some magical property, overworld
+        },
+        "blue_geode_ore":{
+            "crystal":200,
+            "weird":15,
+            //cheaper diamond like item with some magical property, overworld
+        },
+        "red_geode":{
+            "crystal":200,
+            "weird":15,
+            "decay":5,
+            "molten":5,
+            // stronger version of the geode before it, found in nether
+        },
+         },
+        "red_geode":{
+            "crystal":200,

```

Comment:
You repeated the same item here, duplicating entries has no effect, only the last one defined will actually be used.